src/java/htsjdk/samtools/util/CodeUtil.java

```diff
@@ -0,0 +1,14 @@
+package htsjdk.samtools.util;
+
+/**
+ * Miscellaneous util methods that don't fit anywhere else.
+ */
+public class CodeUtil {
+
+    /** Mimic of Oracle's nvl() - returns the first value if not null, otherwise the second value. */
+    public static <T> T nvl(final T value1, final T value2) {
```

Comment:
Didn't we talk about giving this a more expressive name than 'nvl', despite its mimicry of Oracle nvl?